config/initializers/lograge.rb

```diff
@@ -0,0 +1,11 @@
+# frozen_string_literal: true
+
+Rails.application.configure do
+  config.lograge.enabled = true
+  config.lograge.custom_options = lambda do |event|
+    exceptions = %w[controller action format id]
+    {
+      params: event.payload[:params].except(*exceptions),
+    }
+  end
```

Comment:
Si je comprends bien on ne log pas ces params car ils n'apportent pas grand chose par rapport au path, c'est ça ? Peut-on ajouter un commentaire (en français si c'est plus clair) pour expliquer le pourquoi ?

Reply:
Oui, c'est l'idée de ce que je comprends. La proposition vient de la doc de la gem pour info : https://github.com/roidrage/lograge#what-it-doesnt-do